ksqldb-engine/src/test/java/io/confluent/ksql/schema/ksql/inference/DefaultSchemaInjectorTest.java

```diff
@@ -299,7 +328,7 @@ public void shouldInjectForCsStatement() {
             + "`mapField` MAP<STRING, BIGINT>, "
             + "`structField` STRUCT<`s0` BIGINT>, "
             + "`decimalField` DECIMAL(4, 2)) "
-            + "WITH (KAFKA_TOPIC='some-topic', KEY_FORMAT='protobuf', KEY_SCHEMA_ID=18, VALUE_FORMAT='avro', VALUE_SCHEMA_ID=5);"
+            + "WITH (KAFKA_TOPIC='some-topic', KEY_FORMAT='protobuf', VALUE_FORMAT='avro');"
```

Comment:
Not sure if I fully understand the test. If I understand correct, we test the case that the user did provide both key and value schema-id. So why are they removed? (The code above change the statement to replace the plain schema-id with the full "raw schema"...)

Seems I am missing something.

In the end, the result statement is what we write into the command topic, right? So should it not contain the user specified schema-ids? Looking in the next PR, it add `KEY_AVRO_SCHEMA_FULL_NAME` and `VALUE_AVRO_SCHEMA_FULL_NAME` but I am wondering why we don't have both -- IDs and NAMEs -- in the end?

Reply:
Ah. I think I mixed something up. Thanks for clarifying.